lib/archethic/transaction_chain.ex

```diff
@@ -1029,7 +1033,26 @@ defmodule Archethic.TransactionChain do
           {:ok, binary()} | {:error, :network_issue} | {:error, :does_not_exist}
   def fetch_first_transaction_address_remotely(address, nodes)
       when is_binary(address) and is_list(nodes) do
-    case P2P.quorum_read(nodes, %GetFirstTransactionAddress{address: address}) do
+    conflict_resolver = fn results ->
+      results
+      |> Enum.reduce(%NotFound{}, fn
```

Comment:
To filter the `%NotFound` you can use the same way as it is done in beacon chain `fetch_summaries_aggregate` :
```elixir
conflict_resolver = fn results ->
  # Prioritize results over not found
  case nil <- Enum.find(results, &match?(%FirstTransactionAddress{}, &1)) do
    nil -> %NotFound{}

    res ->
      Enum.min_by(res, &DateTime.to_unix(&1.timestamp, :millisecond))
  end
end
```

Reply:
I think you should also discard the `DateTime.from_unix!(0)`

Reply:
> I think you should also discard the `DateTime.from_unix!(0)`

This would never happen for the first transaction message. When there is no transaction in the chain of the requested address, it return a NotFound message